static/js/components/ValidationAlert.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+
+import type { ValidationErrors } from '../flow/profileTypes';
+
+export default class ValidationAlert extends React.Component {
+  props: {
+    errors:   ValidationErrors,
+    children: React$Element[],
+  };
+
+  alertMessage: Function = (): React$Element|void => {
+    const { errors } = this.props;
+    if ( !_.isEmpty(errors) ) {
+      return (
+        <span className="message">
+          Please fix the errors above and try again
+        </span>
+      );
+    }
+  };
+
+  render () {
+    const { children } = this.props;
+    return (
+      <div className="validation-alert">
+        { this.alertMessage() }
+        <div className="actions">
+          { children }
+        </div>
+      </div>
```

Comment:
OK cool, that gives me a better idea of what you're suggesting.

Something occurs to me though, which is that we might want to use the validation alert in places other than in dialogs, in particular on the personal info page. If someone is on a device without much vertical room (say, a smartphone in landscape mode or a netbook) they might not see the validation errors up top when they click save, the `ValidationAlert` component as written can be just dropped into the personal tab component.

Thoughts?

Here's an example of what it looks like:

![validation](https://cloud.githubusercontent.com/assets/6207644/17257384/d87ead56-558f-11e6-9efd-3526cfb3e1a2.png)